client/src/components/Icons/Icon.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import SVGIcon from './SVGIcon';
+import './SVGIcon.css'
+
+
+storiesOf('SVGIcon', module)
+    .add('Logo', () => 
+    <div className='icon-container'>
+        <div className='logoNav'>
+            <SVGIcon className='icon' name='location'/>
+            <SVGIcon className ='icon' name='email' />
+            <SVGIcon className ='icon' name='phone'/>
+            <SVGIcon className ='icon' name='instagram'/>
+        </div>
+        <div className='logoNav'>
+            <SVGIcon className ='icon' name='calendar' fill = 'black'/>
+            <SVGIcon className ='icon' name='location' fill = 'black'/>
+            <SVGIcon className ='icon' name='time' fill = 'black'/>
+        </div>
+    </div>
+    )
```

Comment:
This is cool, but we want to keep this clean and focused just on rendering components in the storybook sandbox. Can you remove the `icon-container` div and `logoNav` div and just render instances of your Icon component?